orchestration/src/main/java/com/sap/ai/sdk/orchestration/Grounding.java

```diff
@@ -65,8 +84,10 @@ public GroundingModuleConfig createConfig() {
     val groundingConfigConfig =
         GroundingModuleConfigConfig.create()
             .inputParams(List.of("userMessage"))
-            .outputParam("groundingContext")
-            .addFiltersItem(filter);
+            .outputParam("groundingContext");
+    groundingConfigConfig.setFilters(
+        Objects.requireNonNullElseGet(filters, () -> List.of(DEFAULT_FILTER)));
```

Comment:
(Minor)

I think you could use the fluent accessor `.filters(...)` directly right? To make it consistent with the other fields `.inputParams(...)` and `.outputParams(...)`.

You could improve code quality a little, by setting `this.filters` as `@Nonnull` by just initializing the variable not with `null` implicitly, but with `List.of(DEFAULT_FILTERS)` explicitly.

Sorry about the back-and-forth.